sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -361,7 +342,25 @@ for (TextSentiment textSentiment : textAnalyticsClient.analyzeSentiment(text).ge
 ```
 
 ## Troubleshooting
-## General
+### General
+Text Analytics clients raise exceptions. For example, if you try to detect the languages of a batch of text with same 
+document IDs, 400 error is return that indicating bad request. In the following code snippet, the error is handled 
```

Comment:
nit: 400 -> `404`

Reply:
404 is an interesting HTTP error code for "Bad Request'. Traditionally, 400 is bad request... and 404 is "Not found". (ie. this URL you made does not exist).

Reply:
https://en.wikipedia.org/wiki/HTTP_404

Reply:
oh mistyped I mean 400 itself just syntax highlighted 🙆